gaffer-worker/src/main/java/uk/gov/gchq/gaffer/Graph.java

```diff
@@ -0,0 +1,57 @@
+/*
+ * Copyright 2020 Crown Copyright
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package uk.gov.gchq.gaffer;
+
+import javax.validation.constraints.NotBlank;
+import javax.validation.constraints.NotNull;
+import javax.validation.constraints.Pattern;
+
+public class Graph {
```

Comment:
Hi,
We're currently looking at fixing this. The CRD requires attributes like kind, spec etc as JSON. By using the GraphConfig object, we can't give the CRD what it needs as GraphConfig doesn't take those values. Looking at the documentation we have also noticed a Graph object, is this what you meant instead or should we still look for a way to incorporate GraphConfig?

Reply:
No I meant using the GraphConfig. The spec is just values to provide to the values.yaml file at the moment so and one of the values (graph.config) maps to something which looks a lot like a GraphConfig object. So it would make sense to re-use the object here.

Reply:
Just to confirm then. Would you like Kai (or other client) to send a requestbody that maps to GraphConfig object for the 'create graph' endpoint when a User wants to create a graph? Or would you like a new Request objects that belongs to gaas-ui 

This Graph object is the request to gaas-rest create graph endpoint so can rename it and reference that it is a request object. That way gaas-ui has some control over the validation and handling incoming requests and can include additional values if required in future.